tests/unit/utils/flow-manager.test.helpers.ts

```
@@ -0,0 +1,138 @@
+    steps: Record<string, unknown>[],
+    name?: string,
+    description?: string
+  ): Record<string, unknown> {
+    return {
+      id,
+      ...(name && { name }),
+      ...(description && { description }),
+      steps,
+    };
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)